src/arrays/ladderops.jl

```diff
@@ -16,11 +16,11 @@
     lowermatrix(lens, particle) = eye_sandwich(lens, particle, lower_single(lens[particle]))
 
     laddercoeffs(n) = sqrt(linspace(1, n, n))
-    lower_single(n) = sparse([1:n-1], [2:n], laddercoeffs(n-1), n, n)
-    raise_single(n) = sparse([2:n], [1:n-1], laddercoeffs(n-1), n, n)
+    lower_single(n) = sparse([1:n-1;], [2:n;], laddercoeffs(n-1), n, n)
+    raise_single(n) = sparse([2:n;], [1:n-1;], laddercoeffs(n-1), n, n)
```

Comment:
I learned about it only by accident. Seems that the more individual names are nicer (e.g. `QuVector` is only printed as `QuArray`). So let's leave it like that then.